<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Shakkikello</class>
 <widget class="QMainWindow" name="Shakkikello">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shakkikello</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Switch">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Switch2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch player</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="Progress1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="Progress2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>START GAME</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Fast">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>157</y>
      <width>56</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>120s</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Slow">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>160</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="START">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>56</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="STOP">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>56</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
